packages/models-library/src/models_library/authentification.py

```diff
@@ -0,0 +1,9 @@
+from enum import auto
+
+from .utils.enums import StrAutoEnum
+
+
+class TwoFAAuthentificationMethod(StrAutoEnum):
```

Comment:
MINOR: the A from 2FA means already authentication. I would rename to avoid double `AA`